src/public/modules/forms/services/form-api.client.factory.js

```diff
@@ -78,6 +78,7 @@ function FormApi($resource, FormErrorService, FormFields) {
 
   // accessMode is either adminForm or publicForm
   let resourceUrl = '/:formId/:accessMode'
+  const V3_PUBLICFORM_URL = ':/formId'
```

Comment:
@karrui did this for visibility rather than just changing '/:accessMode' to be `''` since we are going to port over all these to `react-query` eventually.